deploy/olm-catalog/apicast-operatorsource.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorSource
 metadata:
-  name: apicast-operator-master-source
+  name: apicast-community-operator-source
```

Comment:
No need to change this operatorsource file. We are not pushing to apicastoperatorcommunity, are we? The less changes, the better.

Reply:
:+1: